pkg/actions/nodegroup/get.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 func (m *Manager) GetAll() ([]*manager.NodeGroupSummary, error) {
-	summaries, err := m.stackManager.GetUnmanagedNodeGroupSummaries("")
+	summaries, err := m.stackManager.GetUnmanagedNodeGroupSummaries("", m.clientSet.CoreV1().Nodes())
```

Comment:
I don't feel overly strongly about this but:

It would be nice if the `stackManager` package just did `cloudformation` calls, how would you feel about moving the kubernetes API calls into this func? I just think it reduces the scope of what the `stackManager` package does, which is already very large :smile: 

We definitely do kuberntes calls in the `stackManager` package elsewhere, but I think aiming for "this package just makes cloudformation calls" would be nice.

WDYT?

Reply:
I agree - I'll refactor this to move the entire `GetUnmanagedNodeGroupSummaries` function to this package as a helper func, agree that we should only use `stackManager` for the cloudformation calls!

Reply:
:ok_hand:

Reply:
I've been making some changes to move `GetUnmanagedNodeGroupSummaries` out of the `stackManager` but it's quite tightly coupled with the other `stackManager` methods. In `GetUnmanagedNodeGroupSummaries`, the Kube API call is part of a loop that iterates over all unmanaged nodegroups, so it would require another loop if it were to be moved by itself, which wouldn't be very efficient. So I think it's worth creating an issue to refactor `GetUnmanagedNodeGroupSummaries` and its `stackManager` methods. What do you think?

Reply:
Moved `kubewrapper.GetNodegroupKubernetesVersion()` to this function in https://github.com/weaveworks/eksctl/pull/3810/commits/47d6e735a51613a5519eb2702f1cc331de0896c5!